### PR TITLE
<AtlasAddressInput/> - new component

### PR DESCRIPTION
**Pull request by gendeld**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6428

File: src/AtlasAddressInput/test/AtlasAddressInput.private.uni.driver.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  const getItemDriverAt = async index => {
+    const optionDrivers = await dropdownLayoutDriver.options();
+    const item = await optionDrivers[index].element();
+    return addressInputItemDriverFactory(item);
+  };
+
+  return {
+    ...publicDriverFactory(base, body),
+
+    /** Returns item prefix at given index
```
Comment (EgoziE) [inline src/AtlasAddressInput/test/AtlasAddressInput.private.uni.driver.js:17]:
```suggestion
    /**
     * Gets item prefix at given index
```

File: src/AtlasAddressInput/test/AtlasAddressInput.private.uni.driver.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+     * @param {number} index
+     * @returns {Promise<node>} Item prefix
+     */
+    getItemPrefixAt: async index => (await getItemDriverAt(index)).getPrefix(),
+
+    /** Returns item suffix at given index
```
Comment (EgoziE) [inline src/AtlasAddressInput/test/AtlasAddressInput.private.uni.driver.js:23]:
```suggestion
    /** 
    * Gets item suffix at given index
```

File: src/AtlasAddressInput/AtlasAddressInput.uni.driver.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export { addressInputDriverFactory as atlasAddressInputDriverFactory } from '../AddressInput/AddressInput.uni.driver';
```
Comment (EgoziE) [inline src/AtlasAddressInput/AtlasAddressInput.uni.driver.js:1]: I understand that the driver is identical, but let's write the whole thing in order to make it easy to add specific functions

File: src/AddressInput/AddressInput.uni.driver.js [hunk 4]
```diff
@@ -82,6 +89,28 @@ export const addressInputDriverFactory = (base, body) => {
      * Clicks on input
      * @returns {Promise<void>}
      */
-    clickInput: inputDriver.click,
+    clickInput: () => inputDriver.click(),
+
+    /**
+     * Returns amount of dropdown items
```
Comment (EgoziE) [inline src/AddressInput/AddressInput.uni.driver.js:95]:
```suggestion
     * Gets amount of dropdown items
```

File: src/AddressInput/AddressInput.uni.driver.js [hunk 5]
```diff
@@ -82,6 +89,28 @@ export const addressInputDriverFactory = (base, body) => {
...
+     * @returns {Promise<number>}
+     */
+    getAmountOfItems: () => dropdownLayoutDriver.optionsLength(),
+
+    /**
+     * Get item main label text at given index
```
Comment (EgoziE) [inline src/AddressInput/AddressInput.uni.driver.js:101]:
```suggestion
     * Gets item main label text at given index
```

File: src/AddressInput/AddressInput.uni.driver.js [hunk 6]
```diff
@@ -82,6 +89,28 @@ export const addressInputDriverFactory = (base, body) => {
...
+     * @param {number} index
+     * @returns {Promise<string>}
+     */
+    getItemMainLabelAt: async index =>
+      (await getItemDriverAt(index)).getMainLabelText(),
+
+    /**
+     * Get item secondary label text at given index
```
Comment (EgoziE) [inline src/AddressInput/AddressInput.uni.driver.js:109]:
```suggestion
     * Gets item secondary label text at given index
```

**Review by EgoziE — APPROVED** [review]
only small changes, overall approved.

**Pull request merged**